src/core/DTNHost.java

```diff
@@ -537,5 +537,29 @@ public boolean equals(DTNHost otherHost) {
 	public int compareTo(DTNHost h) {
 		return this.getAddress() - h.getAddress();
 	}
+	/**
```

Comment:
This feature does not sound to me generic enough for the DTNHost class. Perhaps it's better implemented in the routing module.